integration-tests/js-compute/build-one.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if [ "$#" -lt 1 ]; then
+  echo "Usage: $0 <test>"
+  exit 1
+fi
+
+test="$1"
+
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+npm install -s
+npm run -s build:test --test="$test"
+
+cd fixtures/"$test"
+fastly compute pack --verbose --wasm-binary "./${test}.wasm"
```

Comment:
Great catch! I'll remove that step.

Reply:
I just remembered that we'll probably need this for when we're running tests on `c@e` infrastructure, so I might leave it in for now if it's not too noisy.

Reply:
Based on Jake's #108, I gather we don't need the `pack` command for deploying to C@E either, because e.g. `fastly compute deploy` should do it automatically.

Even if I'm mis-understanding that: as far as I can tell, doing it here doesn't serve a purpose for us right now, so I think we should drop it until we need it.